### DISCUSSION TITLE
Better font for Persian & Arabic & Kurdish

### DISCUSSION THREAD

**Discussion by ShahriarKh** [monkeytypegame/monkeytype#2816] [Ideas] [answered]
😑 Current font:
![image](https://user-images.githubusercontent.com/31452340/162410632-25f7179a-c7b2-4881-9ac2-a26c1d01faf7.png)

😺 My recommendation is: Vazir Code (monospaced)
https://github.com/rastikerdar/vazir-code-font
![image](https://user-images.githubusercontent.com/31452340/162410947-49dd14fd-8289-4e6e-b16f-d55259763a4b.png)

😺 or Vazir Matn:
https://github.com/rastikerdar/vazirmatn
![image](https://user-images.githubusercontent.com/31452340/162410986-4d95e02e-06f0-48f7-972a-a40fc32ca506.png)

Vazir matn is free, open-source and support a large range of glyphs (Persian, Arabic, Kurdish, ...)

**Comment by Miodec** [comment, marked as answer]
Have you tried other fonts that are already on the website? Are they all bad with Arabic?

**Reply by ShahriarKh** [reply]
**Lalezar** and **Noto Naskh Arabic** are better than nothing, but Vazir matches the minimal style of the website more than these two. Also, what I'm suggesting is a default font for Persian, Arabic, ..., not just another font in the font selector.
Like this:
```css
--font: "Roboto Mono", "Vazirmatn";
```

**Reply by Miodec** [reply]
Alright, ill add it.

**Reply by ShahriarKh** [reply]
Great.
